README.md

```diff
@@ -102,6 +102,20 @@ When dispatching a `SET` action with one or more data items, the `id` of each
 data item will be used as the key in Redis, possibly prefixed with the `prefix`
 option and the `type` (if `useTypeAsPrefix` is `true` or not set).
 
+#### Pinging Redis
+
+To send a [PING command](https://redis.io/docs/latest/commands/ping/) to Redis,
+dispatch a `SERVICE` action with type `ping`:
+
+```javascript
+const response = await great.dispatch({
```

Comment:
It's great to get this exposed, because we intend to start using it to try to keep the connection open by pinging server every `redisConnectionTimeout - x seconds` seconds